tests/Unit/EventTest.php

```diff
@@ -154,6 +154,35 @@ public function test_cron_life_time(): void
         );
     }
 
+    public function test_cron_life_time_getters(): void
```

Comment:
Split this test into two tests, one per method.

Reply:
Yes sir :)

Reply:
Done